palantir-java-format-benchmarks/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'distribution'
+apply plugin: 'me.champeau.jmh'
+
+configurations {
+    consumingNativeImage
+    consumingPalantirJars
```

Comment:
Not sure `consuming`/`consumable` add much here - I'd just call them something like `formatterNativeImage` and `formatterNativeJars`.